OracleDatabase/SingleInstance/dockerfiles/21.3.0/runOracle.sh

```diff
@@ -111,6 +111,24 @@ trap _int SIGINT
 # Set SIGTERM handler
 trap _term SIGTERM
 
+# Observer only flow
+if [ "${OBSERVER_ONLY}" = "true" ]; then
+   export OBSERVER_BASE_DIR=${ORACLE_BASE}/oradata/
+   $ORACLE_BASE/$CREATE_OBSERVER_FILE $OBSERVER_NAME $PRIMARY_DB_CONN_STR $ORACLE_PWD $OBSERVER_BASE_DIR || exit 1;
+   if test [ ! -f "$OBSERVER_BASE_DIR/$OBSERVER_NAME/observer.log" ] then
```

Comment:
semicolon is missing after " test [ ! -f "$OBSERVER_BASE_DIR/$OBSERVER_NAME/observer.log" ] " , please checl

Reply:
Addressed in the following commit:
https://github.com/yunus-qureshi/docker-images/pull/18/commits/9a80cee7fb0d350fce054fef9bac3c9096e5a5a9